tests/unit/test_matrix.py

```diff
@@ -229,3 +229,16 @@ def and_its_blocks_know_their_internal_representation(self, request):
         blocks = _BaseSubtotals.blocks(fake_cube, fake_measure)
         _blocks.assert_called_once()
         assert blocks == fake_blocks
+
+    def it_knows_how_to_assemble_its_blocks(self, request):
+        bs = _BaseSubtotals(None, None)
```

Comment:
Even if it ends up causing line wrapping, I generally find it most readable to derive an instance name directly from the classname, like `base_subtotals` as an instance of `_BaseSubtotals` in this case. Since this is a base-class, `subtotals` might be apt, like "an instance of any subclass of `_BaseSubtotals`.

Reply:
Also, I'm not sure I remember exactly why, but I've found it best to put the instance construction as the last line of the fixturing, just above the actual call. I suppose it's because often there are other fixture components that are used as parameters in the call and it's nice to always know where to look for that line which appears in almost every unit test.

Reply:
done